ansible/roles/ocp-workload-aiedge-computing-blueprint/tasks/workload.yml

```diff
@@ -0,0 +1,578 @@
+    - "{{ manuela_gitops_repo_dir }}/config/instances/manuela-stormshift-staging-gcp/machine-sensor/machine-sensor-2-configmap.properties"
+    - "{{ manuela_gitops_repo_dir }}/config/instances/manuela-stormshift-staging-gcp/messaging/route.yaml"
+    - "{{ manuela_gitops_repo_dir }}/config/instances/manuela-tst/line-dashboard-configmap-config.json"
+    - "{{ manuela_gitops_repo_dir }}/config/instances/manuela-tst/line-dashboard-route.yaml"
+    - "{{ manuela_gitops_repo_dir }}/config/instances/manuela-tst/machine-sensor-1-configmap.properties"
+    - "{{ manuela_gitops_repo_dir }}/config/instances/manuela-tst/machine-sensor-2-configmap.properties"
+    - "{{ manuela_gitops_repo_dir }}/config/instances/manuela-tst/messaging-route.yaml"
+    # yamllint enable rule:line-length
+
+- name: Commit changes to the gitops routes
+  shell:
+    chdir: "{{ item }}"
+    cmd: "git commit -a -m 'Update route to use cluster wildcard domain'"
+  loop:
+    - "{{ hub_repo_dir }}"
+    - "{{ edge_repo_dir }}"
+    - "{{ manuela_gitops_repo_dir }}"
+    - "{{ manuela_dev_repo_dir }}"
+  ignore_errors: yes
+
+
+- name: Add gitea user to blueprint secret
+  replace:
+    # yamllint disable rule:line-length
+    path: "{{ hub_repo_dir }}/base/02_cluster-addons/07_tekton/secrets/github.yaml"
+    regexp: 'GITHUBUSER'
+    replace: "{{ ocp_workload_aiedge_computing_blueprint.gitea_user | b64encode }}"
+    # yamllint enable rule:line-length
+
+- name: Add gitea user token to blueprint
+  replace:
+    # yamllint disable rule:line-length
+    path: "{{ hub_repo_dir }}/base/02_cluster-addons/07_tekton/secrets/github.yaml"
+    regexp: 'GITHUBTOKEN'
+    replace: "{{ ocp_workload_aiedge_computing_blueprint.gitea_user_pw | b64encode }}"
+    # yamllint enable rule:line-length
+
+- name: Commit changes to the tekton git secret
+  shell:
+    chdir: "{{ hub_repo_dir }}"
+    cmd: "git commit -a -m 'Update git user auth for tekton pipelines'"
+  ignore_errors: yes
+
+###################################################################################################
+# Deploy infrastructure for the Core DataCenter
+###################################################################################################
+- name: Install Auto Approver kustomize manifest
+  shell:
+    cmd: "oc apply -f '{{ hub_repo_dir }}/base/02_cluster-addons/{{ item }}'"
+  loop:
+    - "01_autoapprover/00_sa.yaml"
+    - "01_autoapprover/01_crb.yaml"
+    - "01_autoapprover/02_approver.yaml"
+
+- name: Install ArgoCD kustomize manifest
+  shell:
+    cmd: "oc apply -k '{{ hub_repo_dir }}/base/02_cluster-addons/03_argocd'"
+  # Retry while we wait for the ArgoCD CRD creation
+  register: result
+  until: result.rc == 0
+  retries: 10
+  delay: 20
+
+- name: Install OpenShift Pipeline kustomize manifest
+  shell:
+    cmd: >-
+      oc apply -k '{{
+        hub_repo_dir }}/base/02_cluster-addons/04_openshift-pipelines'
```

Comment:
Please use `command` rather than `shell` here. Also, this line breaking looks awkward?

Reply:
Updated all `shell` calls to use `command`. That line break has been updated. It was a leftover from my initial attempt to get all of the lines to fit within the 120 char line length